invenio_banners/ext.py

```diff
@@ -22,14 +23,15 @@ def init_app(self, app):
         """Flask application initialization."""
         self.init_config(app)
         app.extensions["invenio-banners"] = self
+        app.jinja_env.globals[
+            "get_active_banner"
+        ] = get_active_banner_for_request
+        app.jinja_env.filters["style_banner_category"] = app.config[
+            "BANNERS_CATEGORIES_TO_STYLE"
+        ]
```

Comment:
Comment: Very minor thought I just had and non-blocking for a merge. 

When it comes to exposing functionality in Jinja, I've found "namespacing" things to be a bit safer in terms of a) avoiding polluting the global context, b) adding extra functionality in the future to a module/extension and making it available in Jinja.

So doing something like:

```python
app.jinja_env.globals['current_banners'] = self  # or a ".proxies.current_banners" LocalProxy
```

and making `get_active_banner` and `style_banner_category` methods of the extension class, makes usage in Jinja a bit cleaner.

Only caveat I see, is that `style_banner_category` is not a Jinja filter anymore, but just a plain function

Reply:
Let's discuss this offiline. Since this is used only in the UI, if you don't mind I will merge this PR and then work on the changes in a new one, so it is smaller.